src/select/__tests__/select-methods.scenario.js

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+
+import {StatefulSelect} from '../index.js';
+import {Button} from '../../button/index.js';
+
+export default function Scenario() {
+  const methods = React.useRef();
+
+  return (
+    <div style={{width: '360px'}}>
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          marginBottom: '20px',
+        }}
+      >
+        <Button
+          onClick={() => {
+            methods.current && methods.current.setDropdownOpen(true);
+          }}
+        >
+          Open Dropdown
+        </Button>
+        <Button
+          onClick={() => {
+            methods.current && methods.current.setDropdownOpen(false);
+          }}
+        >
+          Close Dropdown
+        </Button>
+      </div>
+      <StatefulSelect
+        methods={methods}
```

Comment:
bike shedding on the prop name.. maybe `methodsRef` to suggest that a ref should be provided?

Reply:
Please always be unrestrained with the bike shedding when reviewing my diffs :)

Thanks for the suggestion. I was thinking the same thing actually.